src/pickWithDefault.js

```diff
@@ -0,0 +1,36 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Similar to `pick` except that this one includes a `key: <default>` pair for
+ * properties that don't exist
+ *
+ * **Note:** A property set to undefined will be left untouched.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.22.2
+ * @category Object
+ * @sig [k] -> * -> {k: v} -> {k: v}
```

Comment:
Thanks much for the clarification.  I'll make that change and push the items you've noted.  They can give feedback then.

Yeah my only concern with Sanctuary constructs in ramda signatures is that the front-page documentation should be updated to make users aware of the strong ties.